### ISSUE TITLE
Put rafts in all graph levels

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#417]
So far, all of the levels that have the push puzzle problem #416 have had water in them. If the player had had a raft, then they could have navigated around the obstruction.

This is not likely to always be the case, but remember, we only need the 30 + 30 levels in the experiment to work on the Tower specifically. Perhaps arbitrarily putting a raft object into every dungeon before the room with the block puzzle could solve our problem (for the sake of the experiment) even if it doesn't fully fix the underlying issue.

It would also be nice to have the raft around more, since it is an interesting object, and is currently only in a level from the original game, which is not ideal. At the very least, I feel that any session where the user plays the original level 4 from the game should also insert the raft into the graph generated levels.

**Comment by schrum2** [comment]
I guess we'll put this issue on hold. I'll remove the Need Soon tag but keep it around.

Note that we are going to want to permanently archive the version of the code used in the study in a special branch, but let's wait on creating that until we are certain the code won't change any more.

**Comment by schrum2** [comment]
Remaining work from this issue has moved to the issues #450 and #452
